distribution/docs/src/main/resources/content/_securityFramework/security-rest-intro.adoc

```diff
@@ -73,8 +73,125 @@ To retrieve the subject, the token is sent through Shiro to the STS Realm where
+|          |1          |         |       |Signed Response                 |      |                                    |
+|          |Access     |         |       |In PAOS Response                |      |3                                   |
+|          |Resource   |         |       |                                |      |AuthNRequest                        |
+|          |           |         |       |                               4|      |In SOAP Request                     |
+|          |          6|         |       \---------\      Signed Response |      |                                    |
+|          |    Supply |         |2                |      In SOAP Response|      |                                    |
+|          |   Resource|         |AuthNRequest     |                      |      |                                    |
+|          |           |         |In PAOS Request  |                      |      |                                    |
+|          |           |         |                 |                      |      |                                    |
+|          |           v         |                 |                      |      |                                    |
+|        +-+-------------------------------------------------------------------------------------------------+        |
+|        |                       |                 |                      |      |                           |        |
+|        |                       v                 |                      v      |                           |        |
+|        |                      +------------------+-----------------------------+-+                         |        |
+|        |   SAML ECP Aware     |                SOAP Intermediary                 |                         |        |
+|        |   Secure Client      |cDEF            (CXF Interceptor)                 |                         |        |
+|        |                      +--------------------------------------------------+                         |        |
+|        |                                                                                                   |        |
+|        +---------------------------------------------------------------------------------------------------+        |
+|                                                                                                                     |
++---------------------------------------------------------------------------------------------------------------------+
+....
+
+SAML ECP can be used to authenticate a non-browser client or non-person entity (NPE).
+This method of authentication is useful when there is no human in the loop, but authentication with an IdP is still desired.
+The IdP Handler will send a PAOS request as an initial response back to the Secure Client, assuming the client has sent the necessary HTTP headers to declare that it supports this function.
```

Comment:
Do we want to spell out `PAOS` first time?

Reply:
So this is a weird one. I don't think it actually stands for anything. It is actually just SOAP backwards and spelled out it is just "Reverse SOAP" I could say "Reverse SOAP (PAOS)" if you think that'd make it easier to understand?